pycsw/core/metadata.py

```diff
@@ -1381,6 +1381,7 @@ def _parse_iso(context, repos, exml):
             len(md.identification.contact) > 0):
             all_orgs = set([item.organization for item in md.identification.contact if hasattr(item, 'organization') and item.organization is not None])
             _set(context, recobj, 'pycsw:OrganizationName', ';'.join(all_orgs))
+            _set(context, recobj, 'pycsw:Contacts', jsons.dumps(md.identification.contact))
```

Comment:
introduced jsons here because json was not able to parse the contact instance from owslib, instead i can add an explicit property by property conversion (https://stackoverflow.com/questions/3768895/how-to-make-a-class-json-serializable)